setup_cfg_fmt.py

```diff
@@ -356,6 +356,13 @@ def format_file(
                 f'\n{LICENSE_TO_CLASSIFIER[license_id]}'
             )
 
+    if cfg.has_section('options'):
```

Comment:
any reason not to do it for all sections and options?

Reply:
Sure if it's safe 🤷